mongo_connector/oplog_manager.py

```diff
@@ -668,16 +678,26 @@ def upsert_each(dm):
                             num_failed += 1
                         else:
                             raise
+                    if num % 10000 == 0:
+                        LOG.info("Upserted %d/%d docs from collection %s",
+                                 num + 1, total_docs, namespace)
```

Comment:
This isn't reliable, since documents may be inserted/removed while this is happening, changing the total. This could be confusing if the last line logged here is "Upserted 90/200 docs from collection %s," since it looks like mongo-connector may have missed a large portion of the collection. Clarifying that this is approximate could make this better.